migrations/testresults.go

```diff
@@ -0,0 +1,109 @@
+package migrations
+
+import (
+	evg "github.com/evergreen-ci/evergreen/db"
+	"github.com/mongodb/anser"
+	"github.com/mongodb/anser/db"
+	"github.com/mongodb/anser/model"
+	"github.com/pkg/errors"
+	"gopkg.in/mgo.v2/bson"
+)
+
+const (
+	// tasksCollection is the name of the tasks collection in the database.
+	tasksCollection = "tasks"
+
+	// OldTasksCollection is the name of the old_tasks collection in the database.
+	oldTasksCollection = "old_tasks"
+
+	// testResultsCollection is the name of the testresults collection in the database.
+	testResultsCollection = "testresults"
+)
+
+// testResult is an element of the embedded test_results array in a task.
+type testResult struct {
+	Status    string  `json:"status" bson:"status"`
+	TestFile  string  `json:"test_file" bson:"test_file"`
+	URL       string  `json:"url" bson:"url,omitempty"`
+	URLRaw    string  `json:"url_raw" bson:"url_raw,omitempty"`
+	LogID     string  `json:"log_id,omitempty" bson:"log_id,omitempty"`
+	LineNum   int     `json:"line_num,omitempty" bson:"line_num,omitempty"`
+	ExitCode  int     `json:"exit_code" bson:"exit_code"`
+	StartTime float64 `json:"start" bson:"start"`
+	EndTime   float64 `json:"end" bson:"end"`
+
+	// Together, TaskID and Execution identify the task which created this testResult
+	TaskID    string `bson:"task_id" json:"task_id"`
+	Execution int    `bson:"task_execution" json:"task_execution"`
```

Comment:
probably don't need to tag the json?

I still think we could probably get away with putting this inside of the function scope.

Reply:
done